### DISCUSSION TITLE
Codespace stucked "Connecting..." when is "Setting up your codespace"

### DISCUSSION THREAD

**Discussion by rodrigopintof** [community/community#112435] [Codespaces]
### Select Topic Area

Bug

### Body

Since yesterday I can't connect to my Codespace. I have been using the cs50's code50 codespace since last year without any issue  until yesterday. I can't get connected even trying in Chrome & Firefox browsers in MacOS, VS Code desktop, browsers in Android mobile. I even try everything I found searching in similar discussions solved like connecting through VPN (Express VPN) and nothing works so far. 
In browser Codespace stucked "Connecting..." when is "Setting up your codespace".
In VS Code Desktop is similar, stays in "connecting" status.
In Github shows the codespace linkt to my account, and GithubStatus.com shows no issue.

Please help!! Thanks for reading!

![image](https://github.com/community/community/assets/116110339/1d5bc7f1-aad9-4803-8807-29fa101f81e3)
![image](https://github.com/community/community/assets/116110339/ece74c1d-05dd-4a86-831c-8b7c22767449)
![image](https://github.com/community/community/assets/116110339/e6b48878-e923-40a9-b16b-c7ef4769729b)
![image](https://github.com/community/community/assets/116110339/af29c02b-9f60-4f76-973e-120e6b399406)

**Comment by davetapley** [comment]
I'm seeing this now as well. 

One it stuck:
> Setting up remote connection: Starting codespace...

The other:
> Setting up remote connection: Finishing up...

**Reply by rodrigopintof** [reply]
After 2 days, it fixed itself.
Now works no problem and have been since.
In every forum, and social network I asked, nobody seemed to had the same problem.
It remains a mystery ¿?

**Reply by davetapley** [reply]
Yep, same here. 
I'm going to raise a support ticket.
